app/scripts/modules/core/projects/dashboard/cluster/projectCluster.directive.html

```diff
@@ -45,9 +45,12 @@
           </a>
         </td>
         <td>
-          <a class="heavy" ng-if="application.build.buildNumber" href="{{application.build.url}}" target="_blank">
+          <a class="heavy" ng-if="application.build.buildNumber && !application.build.images" href="{{application.build.url}}" target="_blank">
             <span>#</span>{{application.build.buildNumber}}
           </a>
+          <ul class="list-unstyled" ng-if="application.build.images" ng-repeat="image in application.build.images">
```

Comment:
Heh, you are correct.  My UI skills are showing.  This also gets rid of that extra line in there that was bothering me.  Ill do it for both.  Thanks!
![image](https://cloud.githubusercontent.com/assets/3035661/17559524/2536d95e-5ee4-11e6-80a9-563a1ba1b5ef.png)